client/src/partials/exchange/exchange.js

```diff
@@ -33,15 +36,23 @@ function ExchangeRateController(Session, Dates, Currencies, Rates, $uibModal) {
 
   // start up the module
   function startup() {
-
+    vm.feedback = null;
     // load supported currencies
     Currencies.read().then(function (data) {
       vm.currencies = data;
 
+      // filter out the enteprise currency
+      vm.outCurrencies = vm.currencies.filter(function (currency) {
+        return currency.id !== Session.enterprise.currency_id;
+      });
+      vm.form.id = null;
```

Comment:
@jniles  When I tested the module, there was a lot of that element is not normally displayed above each time you remove all exchange rates